index.d.ts

```diff
@@ -0,0 +1,39 @@
+declare module 'envvar' {
+  /**
+   * Retrieves a given environment variable as a string.
+   *
+   * @throws if the environment variable is not set and a default is not
+   *         specified.
+   * @throws if the value of the environment variable is not a string.
```

Comment:
This line can be removed, since set environment variables are always strings:

```console
$ FOO=42 node --print 'typeof process.env.FOO'
string
```